lib/nonblock.c

```diff
@@ -31,9 +31,6 @@
 #include <fcntl.h>
 #endif
 
-#if (defined(HAVE_IOCTL_FIONBIO) && defined(NETWARE))
```

Comment:
Could you keep this `#include <sys/filio.h>` but change the condition to `#ifdef HAVE_SYS_FILIO_H`? I'm trying to revive support for classic Mac OS and it needs this.

Reply:
Couldn't your patch just as well include it if it needs it? I understand that it'll be a merge conflict once this goes in (if it does) but it should be trivially fixed.

Reply:
OK.